### PR TITLE
[sailfish-components-webview] Use bigger tiles for FullHD and increase pixel ratio. Fixes JB#39557

### PR DESCRIPTION
**Pull request by rainemak**

### CODE REVIEW

**Comment by rainemak** [comment]
@jpetrell @chriadam review ?

Applied pixel ratio for web content must result integer. Hence, those checks.

File: lib/webenginesettings.cpp [hunk 1]
```diff
@@ -76,22 +76,25 @@ void SailfishOS::WebEngineSettings::initialize()
     // Round to nearest even rounding factor
     pixelRatio = qRound(pixelRatio / 0.5) * 0.5;
 
+    int screenWidth = QGuiApplication::primaryScreen()->size().width();
+
     // If we're on hdpi and calcaluted pixel ratio doesn't result integer dimensions, let's try to floor it.
```
Comment (pvuorela) [inline lib/webenginesettings.cpp]: typo :)

**Comment by rainemak** [comment]
updated.

**Comment by pvuorela** [comment]
As talked, still a chance that qRounded ratio gets wrong even if it was earlier right, but shouldn't break anything existing and can be adjusted again later.

LGTM.

**Pull request merged**